plugins/module_utils/mh/base.py

```diff
@@ -24,6 +24,13 @@ def __init__(self, module=None):
         if not isinstance(self.module, AnsibleModule):
             self.module = AnsibleModule(**self.module)
 
+    @property
+    def diff_mode(self):
+        return self.module._diff
+
+    def __getattr__(self, attr):
+        return getattr(self.module, attr)
```

Comment:
I'm not sure this is a good idea. I think it's better to explicitly only provide things we want to support. Otherwise we could easily end up in a situation where adding a new explicit function/attribue to ModuleHelper breaks backwards compatibility because some code relies on that attribute being redirected to `AnsibleModule`.

Reply:
okie, adjusted